.github/workflows/release-version.yml

```
@@ -0,0 +1,108 @@
+# This workflow automatically creates a release PR for the CI integration.
+
+name: Create Release PR
+
+env:
+  GIT_AUTHOR_EMAIL: 'packages@datadoghq.com'
+  GIT_AUTHOR_NAME: 'ci.datadog-ci'
+
+on:
+  workflow_dispatch:
+    inputs:
+      semver:
+        description: 'Semver argument for the version bump to do on the CI integration'
+        default: 'minor'
+        type: choice
+        options:
+          - 'patch'
+          - 'minor'
+          - 'major'
+
+jobs:
+  create-release-pr:
+    runs-on: ubuntu-latest
+    steps:
+      # Do the changes
+      - name: Get GitHub App token
+        id: get-token
+        uses: tibdex/github-app-token@v1
+        with:
+          app_id: ${{ secrets.RELEASE_AUTOMATION_GITHUB_APP_ID }}
+          private_key: ${{ secrets.RELEASE_AUTOMATION_GITHUB_APP_PRIVATE_KEY }}
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          token: ${{ steps.get-token.outputs.token }}
+      - name: Create release branch
+        run: git checkout -b local-branch
+      - name: Set git user
+        run: |
+          git config user.name "${GIT_AUTHOR_NAME}"
+          git config user.email "${GIT_AUTHOR_EMAIL}"
+      - name: Bump CI integration version
+        id: bump-version
+        run: |
+          OLD_VERSION_TAG=$(git describe --abbrev=0 --tags) # e.g. v1.0.0
+          OLD_VERSION=${OLD_VERSION_TAG/v/}
+
+          NEW_VERSION=$(./ci/increment-version.sh ${{ github.event.inputs.semver }})
+          NEW_VERSION_TAG="v$NEW_VERSION" # e.g. v1.1.0
+
+          # Update README with new version
+          sed -i "s/$OLD_VERSION/$NEW_VERSION/" {README.md}
```

This would probably work, but keeping it this way is odd:

```suggestion
          sed -i "s/$OLD_VERSION/$NEW_VERSION/" README.md
```